src/ophys_etl/modules/roi_cell_classifier/compute_classifier_artifacts.py

```diff
@@ -166,44 +166,23 @@ def _write_thumbnails(self,
         mask = np.zeros(max_img.shape, dtype=np.uint8)
         mask[pixel_array[0], pixel_array[1]] = 255
 
-        # Compute center of cutout from ROI bounding box.
-        center_row = ophys_roi.bounding_box_center_y
-        center_col = ophys_roi.bounding_box_center_x
-
-        # Find the indices of the desired cutout in the image.
-        row_indices = self._get_cutout_indices(center_row,
```

Comment:
Looks like you did not remove `_get_cutout_indices` or `_get_padding` from this module. Please do so to avoid code duplication.

Reply:
Huh, guess I must have missed that commit. Could have sworn I did it.